Scala/Scala.sublime-syntax

```diff
@@ -111,6 +226,26 @@ contexts:
       captures:
         1: storage.type.scala
         2: entity.name.type.scala
+      push:
+        - match: '(?=[\n;\}\)\]])'
+          pop: true
+        - match: '\['
+          push:
+            - match: '\]'
+              pop: true
+            - include: delimited-type-expression
+        - match: '(<:|>:)'
+          scope: keyword.operator
```

Comment:
I think there's still a few of these (non-`.scala` scopes) floating around.  I'll try to take a pass through and clean them all up.